content/docs/enterprise/tf/main.tf

```diff
@@ -0,0 +1,40 @@
+provider "kubernetes" {}
+
+provider "random" {}
+
+module "pomerium_enterprise" {
+  source = "git::https://github.com/pomerium/documentation//terraform/enterprise-k8s?ref=v0.28.0"
```

Comment:
It feels weird to have this module in our docs repo.  Maybe put it in pomerium/pomerium?

Reply:
well, that's for the enterprise console - might be little weird to have it in pomerium/pomerium either, no?